### PR TITLE
Add ability to specify compression method and level

### PR DESCRIPTION
**Pull request by hashworks**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -116,6 +119,7 @@
   Integer[0] $keep_weekly                                  = 36,
   Integer[0] $keep_daily                                   = 60,
   Integer[0] $keep_within                                  = 30,
+  String[1] $compression                                   = 'lz4',
```
Comment (bastelfreak) [inline manifests/init.pp:122]: should we turn this into an enum? there is only a small lost of allowed values

Comment (hashworks) [reply manifests/init.pp:122]: No. One can provide different levels of compression like `zstd,10`.

Comment (hashworks) [reply manifests/init.pp:122]: (Also there is stuff like `auto,zstd,5`)

**Comment by bastelfreak** [comment]
fedora is expected to fail.

**Pull request merged**